script/unicode/lakefile.lean

```diff
@@ -0,0 +1,15 @@
+import Lake
+open Lake DSL
+
+package "Unicode" where
+  -- add package configuration options here
+
+lean_lib «Unicode» where
+  -- add library configuration options here
+
+@[default_target]
+lean_exe "generate_tables" where
+  root := `Generation
+
+lean_exe "verify_tables" where
```

Comment:
Depending on how exactly CI builds this, we could consider adding
```suggestion
@[default_target]
lean_exe "verify_tables" where
```
so that we catch compile errors right when we introduce them even if the verify script isn't actually executed.